app/src/main/res/layout/activity_main.xml

```diff
@@ -85,6 +85,8 @@
             android:drawableLeft="@drawable/ic_round_search"
             android:drawablePadding="10dp"
             android:hint="Find wallpaper..."
+            android:imeOptions="actionSend"
```

Comment:
I think w need to use actionSearch instead of actionSend, because our action is search

Reply:
yeah i will change it